swagger.json

```diff
@@ -1022,15 +1047,15 @@
           }
         },
         "responses" : {
-          "200" : {
+          "201" : {
             "description" : "contact created"
```

Comment:
unrelated: can we capitalize `C` in contact?

Reply:
Why not!

Reply:
Actually I'm not sure what to put here. The actual message is `${contact.id} created`. For now I'm going to leave as is, but we'll circle back on all responses like this to figure out what we want to say there.